<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextBrowser" name="tbCommands">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>381</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdClientCount">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>381</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbClients">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Connected Clients:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDelete">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>220</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Entfernen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDisconnect">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnecten</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbUserReload">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>320</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Neu Laden</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUserName">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>420</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePasswort">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>450</y>
     <width>161</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbUsername">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>420</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Benutzername</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbPasswort">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>450</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Passwort</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAgree">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>420</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>BestÃ¤tigen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbExit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>480</y>
     <width>381</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Schliesen</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lwUsers">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lwClients">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbDcReload">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>110</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Neu Laden</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
